vulkaninfo/CMakeLists.txt

```diff
@@ -1,5 +1,81 @@
 # CMakeLists.txt file for building Vulkaninfo
 
+set(VULKANINFO_INCLUDE_DIRS
```

Comment:
I don't think that this is needed.  vulkaninfo really only needs vulkan.h.  This sort of construct is often used to get code-generated include files, which are not being used here.